Automattic/companion
# Add options page for tampering with Jetpack's consntants

#### Testing instructions

1. Launch a regular JN site
2. ssh into the site and `rm -rf plugins/companion`
3. step into the `plugins` dir and clone this branch
    ```console
     git clone -b add/jetpack-constants https://github.com/Automattic/companion
     git checkout add/jetpack-constants
   ```

4. Confirm that you see the settings page

![image](https://user-images.githubusercontent.com/746152/42538172-8a1c7b16-846d-11e8-84c5-23366e114863.png)
